website/src/content/docs/general/FAQ.md

```diff
@@ -38,37 +34,6 @@ Some examples are:
 - Monica
 - Recipes
 
-## Inotify Issues
-
-Linux, including TrueNAS SCALE, has a limited count of "open" or "read" files, that you might need to increase as follows:
-
-### Set Inotify on TrueNAS Scale
-
-System settings > Advanced > Sysctl section. It should be noted that adjusting these values will slightly increase RAM usage, and you need to reboot SCALE for them to take effect after being added. Good values to start with would be:
-
-- `fs.inotify.max_user_watches = 524288`
-- `fs.inotify.max_user_instances = 512`
```

Comment:
Make it linux instead of truenas scale and add section for ClusterTool/Talos clusters stating its already set by default